lib/Alchemy/Phrasea/SearchEngine/Elastic/Indexer/Record/Hydrator/ThesaurusHydrator.php

```diff
@@ -22,12 +22,13 @@
 class ThesaurusHydrator implements HydratorInterface
 {
     private $thesaurus;
+    private $candidate_terms;
     private $helper;
 
-    public function __construct(Thesaurus $thesaurus, CandidateTerms $candidateTerms, RecordHelper $helper)
+    public function __construct(Thesaurus $thesaurus, CandidateTerms $candidate_terms, RecordHelper $helper)
```

Comment:
I know you prefer underscores in method variables but as we tend to go the symfony way, I think it would be better to be consistent and to stick to studlyCaps variable naming.